venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,344 @@
+            else: 
+                print(f'Nothing posted for ARR Submission {arr_submission_forum}')
+                return(submission_output_dict)
+        else:
+            print(f"signature of acl commitment note {acl_commitment_note.id} not in arr submission {arr_submission_forum} authors")
+            return submission_output_dict
+    else:
+        print(f"acl commitment note {acl_commitment_note.id} links to arr submission {arr_submission_forum} with invalid invitation")
+        return submission_output_dict
+            
+        
+    
+        
+# Post blind submission (calls post_reviews)
+def post_blind_submission(acl_submission_id, acl_submission, arr_submission, submission_output_dict, acl_commitment_note):
+    # Post requisite groups 
+    number = acl_submission.number
+    # Create paperX/Authors 
+    authors = openreview.Group(
+        id = 'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number), 
+        signatures = [
+            'aclweb.org/ACL/2022/Conference'
+            ],
+        signatories = [
+            'aclweb.org/ACL/2022/Conference', 
+            'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number)
+            ],
+        readers = [
+            'aclweb.org/ACL/2022/Conference', 
+            'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number)
+            ],
+        writers = [
+            'aclweb.org/ACL/2022/Conference'
+            ],
+        members = acl_submission.content['authorids']
+    )
+
+    authors_posted = client.post_group(authors)
+    assert authors_posted, print('Failed to post author groups: ', acl_submission_id)
+
+    # Create paperX/Conflicts
+    conf_id = arr_submission.invitation.rsplit('/', 2)[0]
+    arr_number = arr_submission.number
+    conflict_members = [
+            '{conf_id}/Paper{number}/Reviewers'.format(conf_id = conf_id, number = arr_number),
+            '{conf_id}/Paper{number}/Area_Chairs'.format(conf_id = conf_id, number = arr_number),
+            'aclweb.org/ACL/2022/Conference/Paper{number}/Authors'.format(number = number)
+            ]
+    author_group = openreview.tools.get_profiles(client, ids_or_emails = authors.members, with_publications=False)
+    
+    #print(acl_submission.content['track'])
+    for SAC in (openreview.tools.get_group(client, "aclweb.org/ACL/2022/Conference/{sac_track}/Senior_Area_Chairs".format(sac_track = sac_name_dictionary[acl_submission.content['track']]))).members: 
+        #print(SAC)
+        SAC_profile = openreview.tools.get_profiles(client, SAC)
+        if(SAC_profile):
+            conflicts = openreview.tools.get_conflicts(author_group, SAC_profile[0])
+            if conflicts: 
+                conflict_members.append(SAC_profile[0].id)
+        else:
+            #print(f'SAC {SAC} does not have an OR profile')
+            conflicts = openreview.tools.get_conflicts(author_group, SAC_profile[0])
```

Comment:
tools.get_profiles returns a dummy profile if the profile doesn't exist, you wouldn't need an else.